webapp/jaasai/views.py

```diff
@@ -91,6 +92,18 @@ def support():
     return render_template("jaasai/support.html")
 
 
+@jaasai.route("/blog/feed")
+def blog_feed():
+    feed_url = "https://blog.ubuntu.com/tag/juju/feed"
```

Comment:
This will need cacheing as blog is slow and could timeout.

Reply:
Could you also use the true feed URL: https://admin.insights.ubuntu.com/tag/juju/feed to save the extra level in the stack.

Reply:
I've updated the feed URL. I think it might be best to add the caching in a second branch if that sounds OK? Rather than make this branch too big (I've filed https://github.com/canonical-websites/jaas.ai/issues/165).